### PR TITLE
pl annual reports

### PR DESCRIPTION
**Pull request by rkimera94**

### CODE REVIEW

File: database/grammar/root.grammar [hunk 1]
```diff
@@ -3,7 +3,7 @@ pldbNode
...
...ampleNode abstractFeatureGrammarLinkNode featuresNode referenceNode replitNode tryItOnlineNode twitterNode typeNode websiteNode fileTypeNode linguistGrammarRepoNode tiobeNode meetupNode webReplNode linkedInSkillNode projectEulerNode hoplNode subredditNode redditDiscussionNode quineRelayNode ubuntuPackageNode codeMirrorNode monacoNode rosettaCodeNode antlrNode originCommunityNode cheatSheetUrlNode isOpenSourceNode indeedJobsNode hackerNewsDiscussionsNode firstAnnouncementNode announcementMethodNode abstractRelationshipNode packageCountNode packageInstallsNode packageRepositoryNode packageAuthorsNode packagesIncludingVersionsNode forLanguagesNode conferenceNode abstractCommonTokenNode languageServerProtocolProjectNode abstractParadigmNode latestReleaseNode nativeLanguageNode jupyterKernelNode keywordsNode stackOverflowSurveyNode annualReportsNode centralPackageRepositoryCountNode gdbSupportNode rijuReplNode esolangNode goodreadsNode dblpNode abstractFlagNode oldNameNode pygmentsHighlighterNode leachim6Node domainNameNode isbndbNode semanticScholarNode githubBigQueryNode compilerExplorerNode pyplNode documentationNode screenshotNode demoVideoNode funFactNode factSponsorsNode photoNode
```
Comment (breck7) [inline database/grammar/root.grammar:6]: weird that @github doesn't highlight the `annualReportsNode` character/word diff here and shows the whole line.

File: database/things/python.pldb [hunk 2]
```diff
@@ -267,6 +267,7 @@ stackOverflowSurvey
   medianSalary 59454
   fans 34929
   percentageUsing 0.48
+annualReports https://www.python.org/psf-landing/
```
Comment (breck7) [inline database/things/python.pldb:270]: great start! this is probably good enough. in the future we may want to link to the actual annual reports by year, but this gets us 80% of the value.

**Comment by breck7** [comment]
Great work @rkimera94 !

File: database/grammar/root.grammar (hunk 1 above)
Comment (rkimera94) [reply database/grammar/root.grammar:6]: I think it was considered to be a line change!

**Pull request merged**